CONTRIBUTING.md

```diff
@@ -16,9 +16,9 @@ Go to a page, click edit, and make your changes, using [markdown](https://guides
 
 When you save it ("make a commit"), there are three fields to fill out. *All of these are optional*, with default values, and in most cases it's fine to leave the defaults.
 
-* **The commit title**: By default it is something like "Update [filename]". You can usually leave this as is (often this is ideal), or you can be more precise, but keep it short.
+* **The commit title**: By default it is something like "Update _filename_". You can usually leave this as is (often this is ideal), or you can be more precise, but keep it short.
```

Comment:
i think we probably want to keep the square brackets as that implies that it's a variable of some kind, more than italics. But for the markdown linter to stop complaining, they probably need to be escaped like this: `\[filename\]`.